app/jobs/send_decisions_analytics_job.rb

```diff
@@ -18,10 +18,10 @@ def query
   end
 
   def file_name
-    @file_name ||= "decisions-analytics_#{date}.csv"
+    @file_name ||= "decisions-data/decisions-analytics_#{date}.csv"
```

Comment:
@d-a-v-e is this directory (bucket) then filename?

Reply:
its `folder/file_name.ext` (the bucket is static and defined as an ENV separately for each environment..just adding them now actually!)